### PR TITLE
perf(db): Improve `get_logs_by_tx_hashes` query

### PR DESCRIPTION
**Pull request by perekopskiy**
## What ❔

Removes ordering by `tx_index_in_block`

## Why ❔

Ordering by `tx_index_in_block` is not needed, (`miniblock_number `, `event_index_in_block`) is enough. We currently have an index on (`miniblock_number `, `tx_index_in_block`, `event_index_in_block`) and it can be removed after this PR is applied.

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.
- [ ] Spellcheck has been run via `zk spellcheck`.
- [ ] Linkcheck has been run via `zk linkcheck`.

### CODE REVIEW

File: core/lib/dal/.sqlx/query-24fa393b7c3a0178806a223ad474e13d46b5ea54a144ec2345a76b865c85f84b.json [hunk 1]
```diff
@@ -1,28 +0,0 @@
...
-        "ByteaArray"
-      ]
-    },
-    "nullable": [
-      false,
-      null
-    ]
-  },
-  "hash": "24fa393b7c3a0178806a223ad474e13d46b5ea54a144ec2345a76b865c85f84b"
-}
```
Comment (RomanBrodetski) [inline core/lib/dal/.sqlx/query-24fa393b7c3a0178806a223ad474e13d46b5ea54a144ec2345a76b865c85f84b.json:28]: why is this deleted?

**Review by RomanBrodetski — APPROVED** [review]

File: core/lib/dal/.sqlx/query-24fa393b7c3a0178806a223ad474e13d46b5ea54a144ec2345a76b865c85f84b.json (hunk 1 above)
Comment (perekopskiy) [reply core/lib/dal/.sqlx/query-24fa393b7c3a0178806a223ad474e13d46b5ea54a144ec2345a76b865c85f84b.json:28]: It was added here https://github.com/matter-labs/zksync-era/commit/22099cb2f90d917a2479b82c8a4165e68c8da878#diff-9a8298d90368acb2dd2e979f9a121d6106ae200c18ce082252bb4b17e693da5e, probably this query was present in some intermediate version of Danil's PR and he didn't rerun `sqlx prepare` after it was removed
